com/ArchiveService.py

```diff
@@ -162,39 +180,50 @@ def log_job_error(msg):
             # check the distance w.r.t the current new stations
 
             StationCode = new_station[0]
-            x   = new_station[1][0]
-            y   = new_station[1][1]
-            z   = new_station[1][2]
+            x = new_station[1][0]
+            y = new_station[1][1]
+            z = new_station[1][2]
             otl = new_station[2]
             lat = new_station[3][0]
             lon = new_station[3][1]
-            h   = new_station[3][2]
+            h = new_station[3][2]
 
             filename = os.path.relpath(new_station[4], repository_data_in)
 
-            tqdm.write(' -- New station %s was found in the repository at %s. Please assign a network to the new '
-                       'station and remove the locks from the files before running again ArchiveService.'
+            tqdm.write(''' -- New station %s was found in the repository
+                       at %s. Please assign a network to the new
+                       station and remove the locks from the files before
+                       running again ArchiveService.'''
                        % (StationCode, filename))
 
             # logic behind this sql sentence:
-            # we are searching for a station within 100 meters that has been recently added, so NetworkCode = ???
-            # we also force the StationName to be equal to that of the incoming RINEX to avoid having problems with
-            # stations that are within 100 m (misidentifying IGM1 for IGM0, for example).
-            # This logic assumes that stations within 100 m do not have the same name!
-            insert_station_w_lock(cnn, StationCode, filename, lat, lon, h, x, y, z, otl)
+            # we are searching for a station within 100 meters
+            #  that has been recently added, so NetworkCode = ???
+            # we also force the StationName to be equal to that of
+            #  the incoming RINEX to avoid having problems with
+            # stations that are within 100 m
+            #  (misidentifying IGM1 for IGM0, for example).
+            # This logic assumes that stations within
+            #  100 m do not have the same name!
+            insert_station_w_lock(
+                cnn, StationCode, filename, lat, lon, h, x, y, z, otl)
 
     elif job.exception:
         log_job_error(job.exception)
 
 
 def check_rinex_timespan_int(rinex, stn):
 
-    # how many seconds difference between the rinex file and the record in the db
-    stime_diff = abs((stn['ObservationSTime'] - rinex.datetime_firstObs).total_seconds())
-    etime_diff = abs((stn['ObservationETime'] - rinex.datetime_lastObs) .total_seconds())
+    # how many seconds difference between
+    #  the rinex file and the record in the db
+    stime_diff = abs((
```

Comment:
this is valid pep8 formatting, but our preference for readability is this style of continuation (when space allows):

```python
     stime_diff = abs((stn['ObservationSTime']
                       - rinex.datetime_firstObs).total_seconds())
 ```
...the above style keeps the first operand (`stn['ObservationSTime']`) on the same line as `abs` call, which provides a stronger visual reminder that we're not just doing plain subtraction.